content/en/guides/hosting/hosting-options/self-managed/kubernetes-operator/operator-airgapped.md

```diff
@@ -30,7 +30,7 @@ Before proceeding with the deployment, you must ensure that the following contai
 * [`docker.io/wandb/controller`](https://hub.docker.com/r/wandb/controller)
 * [`docker.io/wandb/local`](https://hub.docker.com/r/wandb/local)
 * [`docker.io/wandb/console`](https://hub.docker.com/r/wandb/console)
-* [`docker.io/bitnami/redis`](https://hub.docker.com/r/bitnami/redis)
+* [`redis`](https://hub.docker.com/_/redis)
```

Comment:
I think we need to test it before we recommend that. The images from Bitnami are made to work on their Helm Chart.
If we want to recommend the image from the Redis directly, we need to test if the Operator can deploy that using the Bitnami helm chart 😬

Reply:
Let's follow up on Slack. :)